src/platform/silabs/PlatformManagerImpl.cpp

```diff
@@ -39,30 +40,27 @@
 #endif // CHIP_SYSTEM_CONFIG_USE_LWIP
 
 #include "AppConfig.h"
-#include "FreeRTOS.h"
 
 using namespace chip::DeviceLayer::Internal;
 
 namespace chip {
 namespace DeviceLayer {
 
 PlatformManagerImpl PlatformManagerImpl::sInstance;
+
+#if SLI_SI91X_MCU_INTERFACE
 #if defined(TINYCRYPT_PRIMITIVES)
 sys_mutex_t PlatformManagerImpl::rngMutexHandle = NULL;
-#endif
 
-#if defined(TINYCRYPT_PRIMITIVES)
 int PlatformManagerImpl::uECC_RNG_Function(uint8_t * dest, unsigned int size)
 {
-    int res;
-
     sys_mutex_lock(&rngMutexHandle);
-    res = (chip::Crypto::DRBG_get_bytes(dest, size) == CHIP_NO_ERROR) ? size : 0;
+    int res = (chip::Crypto::DRBG_get_bytes(dest, size) == CHIP_NO_ERROR) ? size : 0;
     sys_mutex_unlock(&rngMutexHandle);
 
     return res;
 }
-#endif
+#endif // TINYCRYPT_PRIMITIVES
```

Comment:
I don't think this comment is still accurate since the TINYCRYPT_PRIMITIVES check was removed.

Reply:
It is not remove, it is still wrapped in the same condition, I just cleanup the block by 
removing the `#endif` at line 52 (origin file) and additional `#if defined(TINYCRYPT_PRIMITIVES)` at line 54(origin).

the `#if defined(TINYCRYPT_PRIMITIVES)` block starts at line 52 (updated file) and ends at line 63(updated file)